lib/app/endpoint/terminology.rb

```diff
@@ -88,41 +95,54 @@ def find_param(parameters, name)
 
       def normalize_valueset_coding(parameters)
         if find_param(parameters, 'coding')
-          find_param(parameters, 'coding').to_hash
+          [find_param(parameters, 'coding').to_hash]
         elsif find_param(parameters, 'system') && find_param(parameters, 'code')
-          { 'code' => find_param(parameters, 'code'), 'system' => find_param(parameters, 'system') }
+          [{ 'code' => find_param(parameters, 'code'), 'system' => find_param(parameters, 'system') }]
+        elsif find_param(parameters, 'codeableConcept')
+          find_param(parameters, 'codeableConcept').to_hash['coding']
         else
-          {}
+          []
         end
       end
 
       def normalize_codesystem_coding(parameters, system_param = nil)
```

Comment:
Can these two normalized method merge into one? So many repeated logic.

Reply:
Possibly. They are very similar...but the codesystem and valueset versions of `validate-code` are juuust different enough that there have to be parsing differences in their parameters.

Reply:
for this method, the difference is system parameter vs url parameter (or default). The caller method can pass retrieve that value based on its own logic and pass in that value to this method.

Reply:
True. I thought this comment was on a different function. I can probably merge these two methods using that suggestion, yeah.